phoenix-core/src/main/java/org/apache/phoenix/exception/SQLExceptionCode.java

```diff
@@ -466,8 +466,11 @@ public SQLException newException(SQLExceptionInfo info) {
         "because this client already has the maximum number" +
         " of connections to the target cluster."),
     
-    MAX_MUTATION_SIZE_EXCEEDED(729, "LIM01", "MutationState size is bigger than maximum allowed number of rows"),
-    MAX_MUTATION_SIZE_BYTES_EXCEEDED(730, "LIM02", "MutationState size is bigger than maximum allowed number of bytes"), 
+    MAX_MUTATION_SIZE_EXCEEDED(729, "LIM01", "MutationState size is bigger" +
+            " than maximum allowed number of rows, please turning autocommit on and try it again."),
```

Comment:
For upserts it would be slower to turn autocommit on since we would call commit for every row. 
I think it would be better if this message was .. try upserting rows in smaller batches or using autocommit on for deletes.